app/src/test/java/org/simple/clinic/newentry/PatientEntryScreenLogicTest.kt

```diff
@@ -238,39 +228,9 @@ class PatientEntryScreenLogicTest {
     ))
     verify(patientRepository).allColoniesOrVillagesInPatientAddress()
     verifyNoMoreInteractions(patientRepository)
-    verify(patientRegisteredCount).get()
-    verify(patientRegisteredCount).set(1)
-    verifyNoMoreInteractions(patientRegisteredCount)
     verify(uiActions).openMedicalHistoryEntryScreen()
   }
 
-  @Test
-  fun `when save is clicked and patient is saved then patient registered count should be incremented`() {
```

Comment:
This test also verifies we are saving the ongoing entry. Is there a different test that does this or else we should just remove that part and leave the rest of test. WDYT?

Reply:
There is another test that covers the on going entry and the patient registration. So this should be okay